packages/ezjs_extension/ezjs_extension.0.1/opam

```diff
@@ -0,0 +1,35 @@
+opam-version: "2.0"
+synopsis: "Binding for Chrome and Firefox extension API"
+maintainer: "OCamlPro <contact@ocamlpro.com>"
+authors: "Maxime Levillain <maxime.levillain@ocamlpro.com>"
+license: "LGPL-2.1"
+homepage: "https://github.com/ocamlpro/ezjs_extension"
+bug-reports: "https://github.com/ocamlpro/ezjs_extension/issues"
+depends: [
+  "ocaml" {>= "4.05"}
```

Comment:
```suggestion
  "ocaml" {>= "4.06"}
```
List.init was introduced in 4.06

Reply:
After this, I believe this is good to merge

Reply:
thanks, I pushed this last fix